packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -187,7 +186,11 @@ export class DataStores implements IDisposable {
                     type: CreateSummarizerNodeSource.FromAttach,
                     sequenceNumber: message.sequenceNumber,
                     snapshot: attachMessage.snapshot ?? {
-                        entries: [createAttributesBlob(pkg, true /* isRootDataStore */)],
+                        entries: [createAttributesBlob(
+                            pkg,
+                            true /* isRootDataStore */,
+                            this.runtime.disableIsolatedChannels,
```

Comment:
why do we need this in both the meta data and attributes?

Reply:
oh. this is both for the root and nested channels. but again, we should encode this is the version, rather than every flag it the attributes

Reply:
Why encode it in the version? We will want to add other feature flags to this attributes blob- i.e. disabling of GC. I was thinking the version can specify when to expect those flags or what the default behavior is.

Reply:
Originally I was encoding it in the version only, but then I realized we will have at least one other toggleable feature here at the same time.

Reply:
The "disable GC" one should only be in the "metadata" blob in Container Runtime right. We can disable GC from there itself and data stores need not know about it.

Reply:
True, but I wanted to keep some consistency between the root and the datastores